sgkit/variables.py

```diff
@@ -386,6 +386,15 @@ def _check_field(
     )
 )
 
+ibs_probabilities, ibs_probabilities_spec = SgkitVariables.register_variable(
```

Comment:
Should this be `ibs_probability`? Variables in sgkit are singular, see `call_genotype_probability` for example.

Reply:
Good point, I think `variant_ibs_probability` may be best as there is an IBS matrix per variant.